src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -4,15 +4,8 @@
 
 {% block toolbar %}
     {% if collector.firewall %}
-        {% if collector.token %}
-            {% set is_authenticated = collector.enabled and collector.authenticated  %}
-            {% set color_code = not is_authenticated ? 'yellow' %}
-        {% elseif collector.enabled %}
-            {% set color_code = collector.authenticatorManagerEnabled ? 'yellow' : 'red' %}
-        {% else %}
-            {% set color_code = '' %}
-        {% endif %}
-
+        {% set is_authenticated = collector.enabled and collector.authenticated %}
```

Comment:
As this variable is used only in the `{% if collector.enabled and collector.token %}` condition of the text, maybe the usage can be changed to `collector.authenticated` instead

Reply:
I've made some changes ... but I'm not sure if it's what you were asking for.